src/main/java/cz/cvut/kbss/study/model/RoleGroup.java

```diff
@@ -0,0 +1,46 @@
+package cz.cvut.kbss.study.model;
+import cz.cvut.kbss.jopa.model.annotations.Id;
+import cz.cvut.kbss.jopa.model.annotations.OWLClass;
+import cz.cvut.kbss.jopa.model.annotations.OWLDataProperty;
+import cz.cvut.kbss.jopa.model.annotations.OWLObjectProperty;
+
+import java.net.URI;
+import java.util.Set;
+
+@OWLClass(iri = Vocabulary.s_c_Person)
+public class RoleGroup {
+
+    @Id
+    private URI uri;
+
+    @OWLDataProperty(iri = Vocabulary.s_p_roleGroupName)
```

Comment:
Do we need new data property? How about using same thing as in Institution.name:

```
    @ParticipationConstraints(nonEmpty = true)
    @OWLAnnotationProperty(iri = Vocabulary.s_p_label)
    private String name;
```